### PR TITLE
Test cleanups and fixes

### PR DESCRIPTION
**Pull request by jleclanche**
Re-filing #1733 -- closed by mistake and unable to reopen.

No squashing please!

### CODE REVIEW

**Comment by jleclanche** [comment]
Per recommendation from @lorddaedra I also enabled warnings for everything by default. This has allowed me to fix the tests, they are now green on Django 2.0.

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,57 @@
...
- - "3.3"
- - "3.4"
- - "3.5"
- - "3.6"
+
+python: "3.6"
+
 env:
   matrix:
-   - DJANGO="Django<1.9"
```
Comment (lorddaedra) [inline .travis.yml]: Okay, same comment as in previous pull request: Python 3.3 with Django 1.8 is still supported until September, 2017, you can check it in version history table here https://en.wikipedia.org/wiki/CPython or directly in https://www.python.org/dev/peps/pep-0398/#lifespan ...

I hope this pull request will be merged before September, 2017 :-)))

Comment (jleclanche) [inline .travis.yml]: Me too :) But that sounds unnecessary; it's a lot of extra time/effort to spend on the one and only 3.3 test which will no longer be supported in just 4 months. Requiring 3.4 is very reasonable for allauth to do.

File: Makefile [hunk 2]
```diff
@@ -8,29 +8,10 @@ po:
...
-	( . $(VIRTUALENV)/bin/activate; cd docs; make html )
-
-isort-check:
-	$(ISORT) -c
-
 isort-fix:
 	$(ISORT)
 
-pep8:
-	flake8 --exclude=migrations allauth
```
Comment (lorddaedra) [inline Makefile]: Hmm, changes in Makefile looks as regression, why would you remove that lines?..

**Comment by pennersr** [comment]
Thanks for taking the time to prep this pull request! Dropping 3.3/1.8 is +1 for me...

**Comment by pennersr** [comment]
It seems that standardjs checking is broken. In the qa output I see this:

`standard: Node v4 or greater is required. `standard` did not run.`

**Comment by jleclanche** [comment]
I wish coverage wouldn't spam as much as it does...

**Comment by jleclanche** [comment]
@pennersr standardjs issues fixed; I pulled it into its own tox env.

**Comment by jleclanche** [comment]
@pennersr Should be good to land now. No squashing! :P

**Comment by jleclanche** [comment]
Added fixes for isort 4.2.8, as well as tox.ini merging (timothycrosley/isort#487)

**Comment by jleclanche** [comment]
@pennersr this is not an easy PR to keep rebasing; what's blocking it? can I help?

**Comment by pennersr** [comment]
@jleclanche  Sorry, just me being swamped.

PR is looking good.. the only thing I am not fond of is moving the isort.cfg into the tox.ini -- these two should not be intertwined, for example, it will break my editor setup which isorts on save... could we move that to its original location again?

**Comment by jleclanche** [comment]
@pennersr 👍 i dropped that particular commit then -- it should have been transparent though, merely require a `pip install isort --upgrade`.

**Comment by pennersr** [comment]
Thanks!

**Pull request merged**